static/app/views/monitors/monitors.tsx

```diff
@@ -80,9 +80,13 @@ class Monitors extends AsyncView<Props, State> {
     return `Monitors - ${this.orgSlug}`;
   }
 
-  componentDidMount() {
+  onRequestSuccess(response): void {
     this.props.setEventNames('monitors.page_viewed', 'Monitors: Page Viewed');
+    this.props.setRouteAnalyticsParams({
+      empty_state: response.data.length === 0,
```

Comment:
@davidenwang why not just record the number of monitors in the response? We can figure out if it's empty if the number is 0.

Reply:
hmmm I guess we don't really need to know how many monitors the user is seeing, just mainly focusing on whether or not the state is empty or has at least 1

Reply:
@davidenwang Personally I think the number of monitors they see might impact user behavior. Up to you though.

Reply:
thats fair, i have a meeting tomorrow to go over some of the analytics, I'll mention it there to see if we care to record it or just leaving as is, is fine